docs/src/pages/css-in-js/advanced/advanced.md

```diff
@@ -158,6 +162,41 @@ import { StylesProvider } from '@material-ui/styles';
 </StylesProvider>
 ```
 
+### makeStyles / withStyles / styled
+
+The injection of style tags happens in the **same order** as the makeStyles / withStyles / styled invocations. For instance the color red wins in this case:
```

Comment:
```suggestion
The injection of style tags happen in the **same order** as the makeStyles / withStyles / styled invocations. For instance the color red wins in this case:
```

Reply:
This is great information to add to the docs 🚀

Reply:
The injection "happens" right, not the style tags, so I believe the `s` should remain.

Reply:
Happens is the verb form. You are correct :+1: